lib/bento/core/client.rb

```diff
@@ -34,14 +38,10 @@ def conn
       )
     end
 
-    def dev_mode?
-      Bento.dev_mode || false
-    end
-
```

Comment:
I think we do not need this method as the config for `dev_mode` is already set during Bento configuration.